baselines/fedper/fedper/run_figures.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
```

Comment:
Should we leave everything uncommented?

Reply:
I think we can do so, I also need to update this file quickly to only include one "experiments" config .yaml file. I'll add a header for each specific run and say one can uncomment this section. Unless you think otherwise